### PR TITLE
Document different ways to create a pipeline

### PR DESCRIPTION
**Pull request by rmitsch**
## Goals

Document/emphasize different ways to create pipelines in docs.

## Description

Moved up and slightly modified a snippet describing the usage of `spacy.blank()` and calling the `Language` object directly.

### Types of change

Changes to existing documentation.

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: website/docs/usage/models.md [hunk 1]
```diff
@@ -59,6 +38,31 @@ your data.
...
+If a trained pipeline is available for a language, you can download it using the
+[`spacy download`](/api/cli#download) command as shown above. In order to use
+languages that don't yet come with a trained pipeline, you have to import them
+directly, or use [`spacy.blank`](/api/top-level#spacy.blank):
+
+```python
+from spacy.lang.fi import Finnish
+nlp = Finnish()  # use directly
+nlp = spacy.blank("fi")  # blank instance
+```
```
Comment (adrianeboyd) [inline website/docs/usage/models.md:50]: I think this example will be less confusing if we switch to something that currently doesn't have a pipeline, so not Finnish.

Comment (rmitsch) [reply website/docs/usage/models.md]: Changed to Ukrainian.

File: website/docs/usage/models.md [hunk 2]
```diff
@@ -59,6 +38,31 @@ your data.
...
+
+Initializing the language object directly yields the same result as generating
+it using `spacy.blank()`. In both cases the default configuration for the chosen
+language is loaded, and no pretrained components will be available.
```
Comment (polm) [inline website/docs/usage/models.md:54]: I would explain what a blank pipeline actually does and why you'd want it. Something like:

> A blank pipeline is typically just a tokenizer. You could create a blank pipeline when you only need a tokenizer, when you want to add more components from scratch, or for testing purposes.

File: website/docs/usage/models.md (hunk 1 above)
Comment (adrianeboyd) [reply website/docs/usage/models.md]: Sorry, there's no way for you to know, but `Ukrainian` may be added soon-ish. How about `Yoruba`?

**Pull request merged**